pkg/unittest/test_suite.go

```diff
@@ -422,6 +424,68 @@ func (s *TestSuite) runV3TestJobs(
 	return &result
 }
 
+// we only define the relevant fields of plugin.yaml
+type pluginYaml struct {
+	Name    string `yaml:"name"`
+	Version string `yaml:"version"`
+}
+
+// GetPluginVersion returns the current version of the plugin
+func GetPluginVersion() string {
+	// try multiple possible locations for plugin.yaml
+	possiblePaths := []string{
+		"plugin.yaml",
+		filepath.Join("..", "plugin.yaml"),
```

Comment:
we most likely should have a BuildVersion constant or not as great do an embed

```go
var (
	//go:embed "plugin/folder/plugin.yml"
	plugin embed.FS
)
```

Reply:
Can you expand a bit on how you would do this?

If build version is a constant, it would need to be written to plugin.yaml during the build, which means the plugin.yaml would have changes after a build. Would you update the build version constant manually?

If I understand the embed version correctly, it would mean we have to embed the plugin.yaml in the same directory as the plugin, so in the repo root, since we can't embed files that are in an ancestor directory. Can you describe how you would do this?

Reply:
> Do you need to be precise? At most, is going to be one minor or one patch version difference.

One minor or patch version difference between what? Not sure what you're referring to here.

> I'm not sure how consistent release is, but plugin is updated as part of changelog at least in this example https://github.com/helm-unittest/helm-unittest/commit/ab615e5067621d5c98d8c85eb4356301ee8add08

Exactly, that's why I decided to parse the plugin.yaml, as it seems to be updated when a new release is created.

How do we proceed here? I think right now, just reading the plugin.yaml is the most pragmatic solution, without changing the release process.

Reply:
> I think right now, just reading the plugin.yaml is the most pragmatic solution

If it's embeeded in the binary Yes. If not, we have a functionality that no longer works for those who running helm-unittest as standalone CLI